README.md

```diff
@@ -22,9 +22,46 @@ git clone https://github.com/thunderbird/appointment
 cp appointment/backend/.env.example appointment/backend/.env
 cp appointment/frontend/.env.example appointment/frontend/.env
 cd appointment
+```
+
+Next we need to pull Thunderbird Accounts. If you have a dev copy you can just symlink the folder to accounts. 
+
+Other-wise run:
+
+```bash
+git clone https://github.com/thunderbird/thunderbird-accounts.git accounts
+```
+
+If you're starting fresh with thunderbird-accounts, please review the documentations at that repo about setting it up.
+
+And finally we can run the service in docker:
+
+```bash
 docker-compose up -d --build
 ```
 
+If you're using Thunderbird Accounts for authentication you'll additionally need to create a Client. You can do so by running the following command:
+
+```bash
+docker-compose exec accounts uv run manage.py create_client 'Appointment' 'dev contact' 'noreply@example.org' 'https://example.org' --env_type dev --env_redirect_url 'http://localhost:5173/auth/accounts/callback' --env_allowed_hostnames 'localhost:8080,accounts:8087'
+```
```

Comment:
Taking suggestions on how we make this a little less scary. ATN uses a makefile which is neat but I hate writing makefiles.